### PR TITLE
fix: upgraded management sdk to 1.11.0

### PR DESCRIPTION
**Pull request by vkalta**

### CODE REVIEW

File: packages/contentstack-import/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "dependencies": {
```
Comment (netrajpatel) [inline packages/contentstack-import/package.json:7]: @vkalta Can you please bump the package version?

File: packages/contentstack-utilities/package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   "author": "contentstack",
```
Comment (netrajpatel) [inline packages/contentstack-utilities/package.json:32]: @vkalta Can you please bump the package version?

File: packages/contentstack/package.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
     "@contentstack/cli-launch": "~1.0.13",
```
Comment (netrajpatel) [inline packages/contentstack/package.json:38]: @vkalta Can you please bump the package version?

**Review by netrajpatel — CHANGES_REQUESTED** [review]

**Comment by netrajpatel** [comment]
@vkalta Can you please update the description of this PR ?

**Review by netrajpatel — DISMISSED** [review]

**Review by netrajpatel — APPROVED** [review]

**Review by antonyagustine — APPROVED** [review]

**Review by netrajpatel — APPROVED** [review]

**Pull request merged**